$NetBSD: patch-ab,v 1.2 2010/12/25 09:59:25 obache Exp $

Remove an obsolete check that pointers fit into 32-bit data types.  I
was unable to find any code remaining that depends on this, and at
least one program (dungeon!) works just fine without having had to do
any hashtable coding!

--- glkterm/main.c.orig	2008-03-18 02:32:07.000000000 +0000
+++ glkterm/main.c
@@ -52,10 +52,12 @@ int main(int argc, char *argv[])
         printf("Compile-time error: glui32 is not unsigned. Please fix glk.h.\n");
         return 1;
     }
+/*
     if (sizeof(window_t *) > 4) {
         printf("Compile-time error: Pointers cannot fit in a glui32. Start writing hashtable code.\n");
         return 1;
     }
+*/
     
     /* Now some argument-parsing. This is probably going to hurt. */
     startdata.argc = 0;
